Corona-Warn-App/src/main/res/values/family_tests_strings.xml

```diff
@@ -6,6 +6,10 @@
     <!-- XTXT News about family test subtitle -->
     <string name="family_test_card_subtitle_news">"There is news"</string>
 
+    <!-- Family Consent Screen -->
+    <!-- XTXT: Family Consent Screen: Legal: Title -->
+    <string name="family_consent_legal_title" translatable="false">"Ihr Einverständnis"</string>
```

Comment:
shouldn't all legal strings be in XML files intended for legal strings instead of using `translatable="false"` flag?

Reply:
This is just the title of the screen. We don't have the screen titles in the legal strings as far as I can see.

Reply:
Ah wait, I see what you mean. Didn't see the translatable flag.